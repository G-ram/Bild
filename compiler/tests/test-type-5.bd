fun a(){
  type b = R | S | P | L | K
  return R;
}
