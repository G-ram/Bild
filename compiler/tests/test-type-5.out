Type-Scope Error @ l3
